sdks/python/apache_beam/examples/complete/game/game_stats.py

```diff
@@ -230,7 +229,7 @@ def expand(self, user_scores):
 class UserSessionActivity(beam.DoFn):
   """Calculate and output an element's session duration, in seconds."""
   def process(self, elem, window=beam.DoFn.WindowParam):
-    yield (window.end.micros - window.start.micros)//1000000
+    yield (window.end.micros - window.start.micros) / 1000000
```

Comment:
Why do you need this change?

Reply:
This is something that pycodestyle corrected. I'll put it back.